mantidproject/conda-recipes
# Script for local testing artifacts from run_docker_build_continuumio-mc2

In trying to work through adding the ENABLE_WORKBENCH in the conda package, I wanted to test that my local build artifacts from `run_docker_build_continuumio-mc2.sh` would work. This script uses the same initial docker image and the build artifacts from the output of `run_docker_build_continuumio-mc2.sh` to `conda install ..` and make sure the imports work.

To test:
1. Run `run_docker_build_continuumio-mc2.sh`
2. Locate the name of the build artifacts and change the `ARTEFACTS_ROOT` variable accordingly in `run_docker_test_continuumio-mc2_for_local_build_artifacts.sh`
3. Run `run_docker_test_continuumio-mc2_for_local_build_artifacts.sh` and make sure no error. Final output will be the Mantid version (Ex. my output was `3.13.20190225.1515`)
